docs/en/interfaces/formats.md

```diff
@@ -1020,6 +1020,49 @@ Example:
 }
 ```
 
+To use object name as column value you can use special setting [format_json_object_each_row_column_for_object_name](../operations/settings/settings.md#format_json_object_each_row_column_for_object_name).
```

Comment:
For me it was quite hard to understand the meaning of a setting from example (I'm not used to JSONObjectEachRow format though). Maybe add something like this?
```suggestion
To use object name as column value you can use special setting [format_json_object_each_row_column_for_object_name](../operations/settings/settings.md#format_json_object_each_row_column_for_object_name). Value of this setting is set to the name of a column, that is used as JSON key for a row in resulting object.
```